README.md

```diff
@@ -1,90 +1,103 @@
 
-* Make sure you have [Docker](https://www.docker.com) installed and running.
+1. Download and prepare required packages
+2. Serve the required packages
+3. Build the Docker image
+4. Run the Docker image in a container
 
-* If you are not part of the team that makes releases, you can skip this step:
 
-  * Clone this repo with the _recurse-submodules_ parameter (some of the submodules are private, so it's fine if you can't download some of them, just move to the next step):
-    * git clone --recurse-submodules <repo>
+### Prerequisites
+Make sure you have [Docker](https://www.docker.com) installed and running.
 
-  * If you have already cloned the repo, you can init and fetch the submodule like this:
-    * git submodule init
-    * git submodule update
+### Download and prepare required packages
+Most of the SDKs used by Extender server have licenses that prevent third party redistribution. To efficiently work with the build server it is recommended to package the SDKs and serve them from a private URL. The URL is defined as the DM_PACKAGES_URL environment variable.
 
-* Make sure you have access to the url where to download packages from `DM_PACKAGES_URL`
+The [Dockerfile](./server/docker-base/Dockerfile) lists the actual packages needed per platform:
 
-  * See the [Dockerfile](./server/docker-base/Dockerfile) for what actual packages are needed.
-  * See [defold/scripts/package](https://github.com/defold/defold/tree/dev/scripts/package) folder for scripts how to create these packages.
-  * If you have all files locally, you can serve them locally like so:
-    * export DM_PACKAGES_URL=http://localhost
-    * cd defold/local_sdks && python -m SimpleHTTPServer
+* Clang LLVM: https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L49-L51
+* HTML5
+   * Emscripten: https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L103
+* Windows
+   * Microsoft Visual Studio 2019: https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L103
+   * Windows Kits: https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L169
+* Android:
+   * NDK and SDK: https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L259-L260
+* iOS and macOS
+   * iOS, macOS, Xcode - previous version: https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L475-L485
+   * iOS, macOS, Xcode - latest version: https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L488-L496
 
-### Build
+We have prepared scripts to package the required files. Use the scripts in the [defold/scripts/package](https://github.com/defold/defold/tree/dev/scripts/package) folder to create these packages.
 
-* Build the Extender Docker image by running:
+NOTE: If you only plan to use the extender server to build for a single platform you may remove the setup steps for the other platforms to speed up the build process.
```

Comment:
Not only speed up? If the any platforms are missing, the Dockerfile won't work?

Reply:
What I meant was not to ignore those steps but to remove them. For instance removing everything related to Emscripten (https://github.com/defold/extender/blob/dev/server/docker-base/Dockerfile#L73-L110) from the Docketfile would mean that Emscripten isn't included in the image and you won't be able to build anything for HTML5 right?